ploneorg/core/tests/test_content_types.py

```diff
@@ -60,3 +62,45 @@ def test_adding_homepage(self):
         fti.global_allow = True
         self.portal.invokeFactory('homepage', 'home')
         self.assertTrue(IHomePage.providedBy(self.portal.home))
+
+
+class PloneReleaseTest(unittest.TestCase):
```

Comment:
we don't have any jenkins job that runs these tests I think, I will create a new one: https://github.com/plone/jenkins.plone.org/issues/201